scripts/storage/secondary/setup-sysvm-tmplt

```diff
@@ -45,6 +45,8 @@ if [[ ! $@ =~ ^\-.+ ]]; then
 fi
 
 OPTERR=0
+DISKSPACE=2120000  #free disk space required in kilobytes
```

Comment:
this is hardcoded? Is this same as the total size of the disk?

Reply:
I believe it's the min required free space required on secondary store - the same value has been used in the cloud-install-sys-tmplt script used for seeding the template https://github.com/apache/cloudstack/blob/main/scripts/storage/secondary/cloud-install-sys-tmplt#L53